source/robot_model/include/robot_model/Model.hpp

```diff
@@ -8,6 +8,7 @@
 #include <pinocchio/multibody/data.hpp>
 #include <pinocchio/algorithm/crba.hpp>
 #include <pinocchio/algorithm/rnea.hpp>
+#include <pinocchio/algorithm/joint-configuration.hpp>
```

Comment:
Shouldn't these includes be in the source file, if they aren't being used in the header file? This is a general question to all of you

Reply:
general answer is yes